src/views/forms/StakeForm/index.tsx

```diff
@@ -144,6 +144,7 @@ export const StakeForm = ({ onDone, className }: StakeFormProps) => {
           txHash,
           amount: amountBN.toNumber(),
           validatorAddress: selectedValidator.operatorAddress,
+          defaultValidatorAddress: defaultValidator?.operatorAddress,
```

Comment:
nit: probably fine, since `defaultValidator` shouldn't change atm, but probably safest to included in the dependency array at ln157.

Reply:
thanks just added